dataframe/convert.go

```diff
@@ -125,6 +124,32 @@ func toScalarMaybe(v starlark.Value) (interface{}, bool) {
 	return nil, false
 }
 
+// convert starlark value to a go native datum
+func toNativeValue(v starlark.Value) interface{} {
+	val, ok := toScalarMaybe(v)
+	if ok {
```

Comment:
combine into a one liner:

```suggestion
	if val, ok := toScalarMaybe(v); ok {
	
```